tools/integration_tests/readonly/delete_object_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Provides integration tests for file operations with --o=ro flag set.
+package readonly_test
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/setup"
+)
+
+func ensureFileSystemLockedForDeletion(objPath string, t *testing.T) {
+	err := os.RemoveAll(objPath)
```

Comment:
is there any diff between os.remove and os.removeall. We need to validate both.

Reply:
os.removeall is erasing all the file dir from the given path and os.remove will remove only given object.
I passed exact path of file or dir in removeall so it will work as os.remove only. should I make different  function for this ?